ckine/figures/figure6.py

```diff
@@ -268,7 +268,7 @@ def get_Mut_EC50s():
     """Creates df with mutein EC50s included"""
     x0 = [1, 2., 1000.]
     concentrations = mutData.Concentration.unique()
-    ligand_order = ['WT N-term', 'WT C-term', 'V91K C-term', 'R38Q N-term', 'F42Q N-Term', 'N88D C-term']
```

Comment:
Presumably these names can come directly from the DataFrame. Maybe that would be safer?